test/functional/tool_wallet.py

```diff
@@ -261,9 +261,12 @@ def run_test(self):
         # Warning: The following tests are order-dependent.
         self.test_tool_wallet_info()
         self.test_tool_wallet_info_after_transaction()
-        self.test_tool_wallet_create_on_existing_wallet()
-        self.test_getwalletinfo_on_different_wallet()
-        self.test_salvage()
+        if not self.options.descriptors:
+            # TODO: Wallet tool needs more create options at which point these can be enabled.
```

Comment:
Running the tests on commit 80bb5ca8cf fails.

This commit might need to come first?